daily_notebook/20240701__modelling_bat_source_pressure_respect_to_microphone.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>I think directionality should not be included as part of transmission loss, so best to remove it from this plot.</p><p> </p><p>BUT, directionality is important so I think you can make a separate plot for that. It is useful to be able to read the approx values directly from a figure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/duty-cycle-investigation/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>